### PR TITLE
readthedocs

### PR DESCRIPTION
**Pull request by SarahOuologuem**
Until now, added info, plots, and files for ingestion of 10X Visium & MERFISH. 

**Update**: added info, plots & files for preprocess_spatial and deconvolution_spatial

### CODE REVIEW

File: docs/ingesting_merfish_data/Ingesting_merfish_data_with_panpipes.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+In the `qc.data` folder, the final `MuData` object with computed QC metrics is stored. The `MuData` object without QC metrics is also available and stored in the `tmp` folder. The metadata of the final `Mudata` object is additionally extracted and saved as tsv file, `mouse_brain_cell_metadata.tsv`.
+Using the [provided example yaml file](pipeline.yml), the first rows and columns of the `mouse_brain_cell_metadata` tsv file look as follows: 
+
+|                  | spatial:fov | spatial:volume | spatial:min_x | spatial:max_x | spatial:min_y | spatial:max_y
+----------------------------------------|------------------|----------------------|----------------------|----------------------|------------------------|-----------------------
+110883424764611924400221639916314253469 |	     0         |  432.1414246586824	  |  151.53048381581905	 |  161.5960840769112   |     4264.619631592184	 |   4278.0332319401205	
+
+With the plots in `spatial/ingestion/figures/spatial` you can now decide on cutoffs for filtering. The plots include visualizations of the spatial embeddings, as well as violin plots, and histograms: 
+
+<img src="histograms.mouse_brain.png" alt="drawing" width="950"/>
```
Comment (crichgriffin) [inline docs/ingesting_merfish_data/Ingesting_merfish_data_with_panpipes.md:85]: Write png as links to the github page, this way we don't have to store them on readthedocs. Also improves the build time.
The `?raw=true` at the end of each link is very important
```
<img src="https://github.com/DendrouLab/panpipes-tutorials/blob/sarah_readthedocs/docs/ingesting_merfish_data/histograms.mouse_brain.png?raw=true" alt="drawing1" width="950"/>
<p align="center">
<img src="https://github.com/DendrouLab/panpipes-tutorials/blob/sarah_readthedocs/docs/ingesting_merfish_data/spatial_spatial_n_genes_by_counts.mouse_brain.png?raw=true" alt="drawing2" width="200"/>
<img src="https://github.com/DendrouLab/panpipes-tutorials/blob/sarah_readthedocs/docs/ingesting_merfish_data/violin_obs_n_genes_by_counts_sample_id.mouse_brain.png?raw=true" alt="drawing3" width="200"/>
</p>
```

Comment (crichgriffin) [reply docs/ingesting_merfish_data/Ingesting_merfish_data_with_panpipes.md:85]: Also please write more informative alt text

File: docs/ingesting_merfish_data/spatial_spatial_n_genes_by_counts.mouse_brain.png
Comment (crichgriffin) [inline docs/ingesting_merfish_data/spatial_spatial_n_genes_by_counts.mouse_brain.png:1]: Please reduce the size of images over 250K

**Comment by SarahOuologuem** [comment]
I now fixed the links of the images and also made sure that all images are < 250kb.

**Review by crichgriffin — APPROVED** [review]

**Pull request merged**